src/main/java/de/zalando/aruha/nakadi/domain/ValidationStrategyConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package de.zalando.aruha.nakadi.domain;
+
+import org.json.JSONObject;
+
+public class ValidationStrategyConfiguration {
+    private String strategyName;
+    private JSONObject additionalConfiguration;
+
+    public String getStrategyName() {
+        return strategyName;
+    }
+
+    public void setStrategyName(final String strategyName) {
+        this.strategyName = strategyName;
+    }
+
+    public JSONObject getAdditionalConfiguration() {
```

Comment:
I think it can be at least marked as @Nullable (or to be Optional in a perfect case :-D )